web-client/src/modules/personalData/components/PersonalDataForm/PersonalDataForm.tsx

```diff
@@ -497,14 +519,21 @@ const PersonalDataForm: React.FC<PersonalDataFormProps> = ({
                   : setTermsAndPrivacyAccepted(null)
               }
             >
-              {t('user_data_form.terms_conditions_text')}
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href="https://github.com/reach4help/reach4help/blob/master/CODE_OF_CONDUCT.md"
+              <Trans
+                i18nKey="modules.personal-data.components.PersonalDataForm.terms_conditions_text"
+                values={{
+                  linkText: t(
+                    'modules.personal-data.components.PersonalDataForm.terms_conditions_link_text',
+                  ),
+                }}
               >
-                {t('user_data_form.terms_conditions_link')}
-              </a>
+                {/* eslint-disable-next-line jsx-a11y/anchor-has-content,jsx-a11y/control-has-associated-label */}
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://github.com/reach4help/reach4help/blob/master/CODE_OF_CONDUCT.md"
+                />
+              </Trans>
```

Comment:
@mcverter There's a couple of similar issues within the current i18n file. You can refer to this whenever you need to do complicated interpolations similarly. We should always consider that word orders can be changed from language to another language.